rag-langchain/rag_app.py

```diff
@@ -0,0 +1,89 @@
+from langchain_openai import ChatOpenAI
+from langchain_core.prompts import ChatPromptTemplate
+from langchain_core.runnables import RunnablePassthrough
+from langchain.embeddings.sentence_transformer import SentenceTransformerEmbeddings
+from langchain.callbacks import StreamingStdOutCallbackHandler
+from langchain.text_splitter import CharacterTextSplitter
+from langchain_community.document_loaders import TextLoader
+from langchain_community.vectorstores import Chroma
+
+from chromadb import HttpClient
+from chromadb.config import Settings
+import chromadb.utils.embedding_functions as embedding_functions
+
+
+import uuid
+import os
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-d", "--docs", default="data/fake_meeting.txt")
```

Comment:
Do we want to make it customizable?

Reply:
Can you clarify? What would we want to customize here?

Reply:
it will be nice to make that a volume mounted/env var 
 value - so can use same image with arbitrary data

Reply:
replace with something like below so we can run podman with -e DOCS_PATH=wherever

```
default_docs = os.environ.get('DOCS_PATH', 'data/fake_meeting.txt')
parser.add_argument("-d", "--docs", default=default_docs)
```

Reply:
That is basically how it works already. You can do `podman run ragapp -d <file-path>`. 
Is there a reason to pick env vars over over command line parameters?

Reply:
Bc we create a pod, add the containers (model_services, sample app, vectordb) in it and start the pod (then we restart the sample app again if it fails), so not sure if it's feasable to pass args this way. We can push a fix later btw.

Reply:
I'm happy to change it if it would be better, but its unclear to me why having them in podman run command as env variables instead of command line arguments would make a difference. The envs would still need to be passed to the command on starting the pod a second time.

Reply:
o nvm, you just mean passing them this way at all into the pod might not work, is that right?

Reply:
We pass the envs at creation so when we start, we just execute the start cmd

Reply:
yes, not sure if with the pod works

Reply:
ok, I will change it.

Reply:
resolving as i think we can add cmds without using additional env